snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptSearchRequest.java

```diff
@@ -116,7 +117,12 @@
 		/**
 		 * Enable score boosting using DOI field
 		 */
-		USE_DOI,
+		USE_DOI, 
+		
+		/**
+		 * Match concept descriptions where the description has language membership in on of the provided locales
```

Comment:
Accidentally a letter.

```suggestion
		 * Match concept descriptions where the description has language membership in one of the provided locales
```